tools/README.node_modules.md

```diff
@@ -1,12 +1,10 @@
 # Cockpit Javascript Dependencies
 
-For included Javascript libraries, and build processes
-Cockpit uses Node.js dependencies to process its code. The required Node.JS
-javascript code is packaged in the `node_modules/` directory when distributed.
+Cockpit uses Node.js and [npm](https://www.npmjs.com/) dependencies to build
+its code and for runtime JavaScript libraries.
 
-Cockpit does not use Node.js or any npm packages code at runtime. In addition
-when building straight from a tarball it is also not necessary to have Node.js
-installed.
+Cockpit does not use npm or Node.js at runtime. When building/installing from
+a release tarball it is also not necessary to have Node.js installed.
```

Comment:
Maybe you want to press the point that we *do* use the modules at runtime, though?

Reply:
Hmm, that's what the "and for runtime JavaScript libraries." in the previous paragraph was about. I changed this whole section again, hopefully more clear now?